src/Elastic.Apm.MongoDb/LICENSE

```diff
@@ -1,2 +1,207 @@
-The code in this directory is based on the elastic-apm-mongo project by Vadim Hatsura (@vhatsura),
```

Comment:
This file is generated on every build.

Reply:
I think we should add src/Elastic.Apm.MongoDb/LICENSE to .gitignore as it flags as a change when running `build.bat pack`

Reply:
I thought about it as well, but we do `WriteOnlyWhenDifferent="true"` and locally for me it does not flag it. Is this maybe a line ending issue again? 🤔 

Anyways, I leave it out now, I think if `LICENSE` and `LICENSE_LOCAL` doesn't change then this should not generate a git change. If we see that it happens then I agree it makes sense to ignore it - will do in a follow-up if needed.